zendesk/1.0.0/README.md

```diff
@@ -0,0 +1,50 @@
+  # Keptn Zendesk Integration
+
+  Create Zendesk tickets directly from Keptn.
+  
+  ## Step 1: Gather Zendesk Details
+  1. Make a note of your Zendesk environment ID: `myenvironment` in `https://myenvironment.zendesk.com`
+  2. Generate an API token. Do so at: `https://myenvironment.zendesk.com/agent/admin/api/settings`
+  3. Make a note of the email address of a User or Light user you wish to create tickets from Keptn: eg. `lightuser@example.com`
+    
+  ![keptn secret](assets/1.png)
```

Comment:
As described in https://github.com/keptn-contrib/artifacthub/pull/104 Artifacthub currently doesn't support relative paths. The reason is that when processing packages AH only reads the metadata files. They don't store or serve any other files. Therefore all the relative paths in the PR should be changed to absolute paths. Will also add this fact to the README to avoid future problems.